pyproject.toml

```diff
@@ -84,6 +84,8 @@ tornado = { version = "^6.0", optional = false }
 parlai = { version = "^1.7.0", optional = true }
 torch = { version = "^1.4.0", optional = true }
 
+# jsonschema
```

Comment:
Comment is generally the purpose for having the library. Here it's likely "api validation"?